
%----------------------------------------------------------------------------------------
%	COVER LETTER
%----------------------------------------------------------------------------------------

% To remove the cover letter, comment out this entire block

\clearpage
\begin{flushright}
  Dr. Zhian N. Kamvar\\
  Department of Infectious Disease Epidemiology\\
  Imperial College London, W2 1NY, UK\\
  \today
\end{flushright}
% \recipient{HR Departmnet}{EcoHealth Alliance\\460 West 34th Street - 17th Floor\\New York, NY 10001-2320} % Letter recipient
% \date{\today} % Letter date
% \opening{Dear Dr. Ross, members of the search committee} % Opening greeting
% \closing{Sincerely,} % Closing phrase
% \enclosure[Attached]{curriculum vit\ae{}, references} % List of enclosed documents

\textbf{10X Genomics}\\
6230 Stoneridge Mall Road\\
Pleasanton, CA 94588-3260

% Responsibilities

%     Plan design, develop, and automate test plans against software releases.
%     Work with software engineers and computational biologists to develop tests, define test metrics and troubleshoot/resolve issues.
%     Report test issues and results in a consistent and timely manner.
%     Support troubleshooting of customer issues.
%     Test installations over a range of operating systems.

% Desired Skills and Background

%     BS/MS in computer science, bioinformatics, computational biology, or equivalent.
%     Good knowledge of Python, Bash, Linux, and Git.
%     Experience in designing and developing validation, verification, and regression tests.
%     Experience with issue tracking systems (Jira).
%     2+ years bioinformatics development/testing or equivalentis highly desirable.
%     Experience in bioinformatics applications and DNA sequencing is highly desirable.
%     Experience with continuous integration systems (Jenkins, Travis) is desirable.
%     Experience with image processing tools and algorithms is desirable.
%     Experience in Go, Rust, JavaScript and/or R is a plus.
%     Experience with Amazon Web Services and/or VMWare is a plus.

\vspace{1ex}
To the recruitment team,

\vspace{1ex}

I saw the position for
\href{https://boards.greenhouse.io/10xgenomics/jobs/1769868?gh_jid=1769868#application}{Software
Engineer in Test} via
\href{https://twitter.com/sjackman/status/1189977010908454912?s=20}{Shawn
Jackman on Twitter} I realized that this position is exactly what I've been
looking for. I am a research software engineer at Imperial College London
developing R packages, report templates, and trainings for field
epidemiologists in outbreak scenarios. My background in bionformatic data
analaysis, seven years experience developing R packages for diverse audiences,
discipline in collaborative software engineering practices (TDD, GitHub flow,
and CI), and eagerness to learn and apply new skillsets makes me an ideal
candidate for this position. Having worked as both as a scientist and software
engineer, I am excited for the opportunity to join he 10X genomics team
developing robust and maintainable test suites for automated bioinformatic
pipelines.

\vspace{1ex}

The idea of working as a software engineer in test is quite appealing to me
because this sits at the intersection of two things I've been passionate about
for the past six years: reproducible research and robust software applications.
For the last two years, it has been my job to ensure that software used to
report statistics from epidemiological data was robust and trustworthy---which
is especially important given the fact that it is used in humanitarian
settings. As one of the technical leads on the
\href{https://r4epis.netlify.com}{R4EPIs} project---a collaboration between the
R Epidemics Consortium (RECON) and M\'{e}decins Sans Fronti\`{e}res (MSF)---I
have first-hand experience in working with both developers and users (many of
whom were not proficient in R) to define verification tests against pathologies
unique to epidemiological data. We used continuous integration on Travis and
code review on GitHub to ensure the quality of contributed code.

\vspace{1ex}

I have all of the desired qualifications for this position. I have a doctorate
in computational biology where I developed R tools for population genetic
analysis. My bioinformatics experience comes largely from from my postdoctoral
work where I developed a
\href{https://github.com/zkamvar/read-processing}{Makefile-based SLURM
pipeline} to process genomic variants from paired-end illumina data of 55
fungal genomes. Most recently, I 


To give a more concrete picutre of my skillset, I have been
developing in R and C since 2012 (version control since 2013, tests and
continous integration since 2014) and currently maintain five packages on CRAN
and have contributed to the development of several others. All of these
packages are tested under continuous integration and one
(\href{https://grunwaldlab.github.io/poppr}{poppr}) is featured in
\textgreater500 scientific publications. All of my published papers have the
code and data needed to reproduce the results. My most recent first-author
analytical work, \href{https://peerj.com/articles/4152/}{Kamvar \textit{et al.}
2017}, is fully reproducible in a Docker container under continuous
integration. 

\vspace{1ex}

Perhaps my biggest accomplishment has been the
\href{https://grunwaldlab.github.io/poppr}{poppr R package} for population
genetic analysis. It has \textgreater500 citations and \textgreater85,000
downloads primarily because it was centered around the users. In addition to
the vast amount of documentation, I have maintained
\href{https://groups.google.com/group/poppr}{an active user forum} where I
field questions, feature requests, and bug reports. This constant cycle of
feedback helped me shape the package and its tests to improve the user
experience and confidence.

\vspace{1ex}

As my carreer focus has shifted from investigative research to scientific
software engineering, I believe this project will be immensely beneficial to my
trajectory as a research software engineer. My background in quantitative
science, robust software development, collaborative capacity, and passion for
both equity and fairness in peer review makes me well-suited to join the team
as a software research scientist. Thank you for your time and consideration, I
look forward to hearing from you. 

\vspace{2ex}

Sincerely,

\vspace{5ex}

\textbf{Zhian N. Kamvar}\\
\textit{Attached: curriculum vit\ae{}, references}





% My qualifications: 

%  - Shipping R packages with C backends (poppr)
%  - Worked with data science tools such as docker containers in cloud-based infastructure (Kamvar et al. 2017)
%  - I am an autodidact and will learn new approaches/technologies to acheive my goals.
%  - Since 2013 I have worked openly and collaboratively on open source software and data analyses. 
%  - I have worked in two distributed computing frameworks (SGE and SLURM) and am knowledgable of how to properly use shared resources.

% Why you want to hire me:

%  - I have six years of experience maintaining open source software, both new and
%    legacy code in both R and C.
%  - I know how to work with data scientists to write user stories that allow me
%    to develop packages that are both user-friendly, robust, and reliable.
%  - I have experience working remotely with a diverse team using GitHub flow and 
%    continuous integration.
%  - I know how to learn new skill sets to solve unique challenges (I learned C
%    to address performance issues in R and I learned Make to create a genomic
%    data analysis pipeline).
%  - I have several years experience of collaborative working.
%  - I have strong technical writing skills.





\clearpage
