

%----------------------------------------------------------------------------------------
%	EDUCATION SECTION
%----------------------------------------------------------------------------------------
\vspace{5pt}

I am a \textbf{highly-motivated software engineer} of several published
packages for population genetics/genomics, epidemiology, and reproducible
publishing in the R programming language. As a \textbf{domain expert in
computational biology}, software engineer, and a science communicator, I have
both the specialist and generalist skillsets needed for \textbf{efficient
communication} and collaboration between team members with \textbf{diverse
technical backgrounds}. My goal in software development is to ensure that my
applications are \textbf{robust} and \textbf{accessible} to researchers
regardless of computational skills. Above all, I aim to ensure my software is
usable, efficient, consistent, and correct through best practices in software
development and reproducibility.

\section{Technical Skills}

\cvitem{Project Management:}{Highly experienced (\textbf{10 years}) in \textbf{collaborative development} with Git and GitHub; managing projects, teams, and code review}
\cvitem{}{\textbf{10 years} experience in developing well-tested and \textbf{cross-disciplinary scientific software}}
\cvitem{}{Employs \textbf{problem-forward} approach to addressing researcher needs through iterative feedback and testing}
% \cvitem{}{Practitioner of \textbf{Test Driven Development}, all packages extensively tested (unit, verification, validation, and regression tests)}
\cvitem{}{Developed an easy-to-use \textbf{open source lesson infrastructure} to build \textbf{reproducible data science lessons}}
\cvitem{}{\textbf{Coordinated communications} to hundereds stakeholders in separate sub-communities about the impacts of the new infrastructure}
\cvitem{}{Seamless roll out to \textbf{\textgreater50 source repositories} maintained by \textbf{\textgreater100 volunteers} for lessons that serve \textbf{\textgreater10,000 learners annually}}
\cvitem{Languages:}{\textbf{Expert R developer} of several widely used R packages (\textbf{10 years})}
\cvitem{}{Experienced in creating unique \textbf{Python 3} modules for population genomic simulation analysis pipeline (\textbf{3 years})}
\cvitem{}{Experienced in parsing structured text content in \textbf{XPath} and \textbf{Lua} (\textbf{3 years})}
\cvitem{}{Developed GitHub Actions to work with the GitHub (REST and GraphQL) API using BASH, R, and \textbf{node JS} (\textbf{3 years})}
\cvitem{}{Experience building \textbf{reproducible analysis with CI and Docker} \textit{(Kamvar, 2017)}}
\cvitem{}{Maintained \textbf{reproducible pipelines in Make and BASH} (\textbf{2 years}) for both continuous integration and HPC frameworks}
\cvitem{}{Implemented \textbf{recursive and bit-level operations in C} to vastly improve speed of R code in the poppr package}


% \cvitem{Bioinformatics:}{Experience with both \textbf{Sun Grid Engine (4 years)} and \textbf{SLURM (1 year) HPC frameworks}}
% \cvitem{}{Developed a Makefile and BASH based \textbf{variant discovery pipeline} for paired-end Illumina sequencing data}

\cvitem{Communication:}{\textbf{Coached >100 graduate researchers} to communicate their research to broad audiences in the award-winning radio program, Inspiration Dissemination
(\url{https://blogs.oregonstate.edu/inspiration})}
%{Highly expereinced in \textbf{quickly troubleshooting} and resolving \textbf{>200 forum questions} in user forums for the poppr and adegenet packages ($\sim$1 day turnaround)}
\cvitem{}{Developed and delivered material for \textbf{9 workshops} in \textbf{3 disciplines} that reached \textbf{>300 people}}
\cvitem{}{Published \textbf{19 peer-reviewed publications} including \textbf{7 first-author publications}}

\section{Selected Projects (on GitHub)}

\cvitemwithcomment{Workflows:}{\href{https://github.com/carpentries/lesson-transition}{carpentries/lesson-transition}}{\textbf{automated workflow} to transform Carpentries-style lessons to new infrastructure}
\cvitemwithcomment{R packages:}{\href{https://carpentries.github.io/sandpaper/}{carpentries/sandpaper}}{user interface to \textbf{reproducibly build and deploy} data science lessons}
\cvitemwithcomment{}{\href{https://docs.ropensci.org/tinkr}{ropensci/tinkr}}{transform markdown to XML and back again}
\cvitemwithcomment{}{\href{https://grunwaldlab.github.io/poppr/}{grunwaldlab/poppr}}{analysis of populations with mixed reproductive modes}
% \cvitemwithcomment{}{\href{https://github.com/thibautjombart/adegenet/\#readme}{thibautjombart/adegenet} (maintainer)}{multivariate analysis of population genetics}
% \cvitemwithcomment{}{\href{https://r4epi.github.io/apyramid}{r4epi/apyramid} (maintainer)}{visualize population pyramids aggregated by age}
% Can't assume links will work. 
\cvitemwithcomment{Analyses:}{\href{https://github.com/everhartlab/sclerotinia-366}{everhartlab/sclerotinia-366}}{\textbf{fully automated} and reproducible analysis in \textbf{Docker}}
\cvitemwithcomment{}{\href{https://github.com/zkamvar/clonal-inference-simulations}{zkamvar/clonal-inference-simulations}}{simulation-based analysis in \textbf{Python}, \textbf{R}, and \textbf{BASH}}
% \cvitemwithcomment{}{\href{https://github.com/zkamvar/read-processing}{zkamvar/read-processing}}{variant discovery from 55 \textit{Sclerotinia sclerotiorum} genomes in \textbf{Make}}

\newpage

\section{Collaborations}

\cventry{2021--Present}{Maintainer}{\href{https://docs.ropensci.org}{the tinkr R package}}{rOpenSci}{}{
  Collaboration with MaÃ«lle Salmon (original author) of rOpenSci to create a
  lightweight R package to parse and transform Markdown documents to XML. This
  has been used in \textbf{validation} of markdown elements and facilitating
  \textbf{automated human language translation}.
}

\cventry{2018--2019}{Lead Developer}{\href{https://r4epis.netlify.com}{R4Epis Project}}{M\'{e}decins Sans Fronti\`{e}res (Doctors Without Borders)}{}{
  Collaborated with \textbf{diverse team of developers and field epidemiologists} to create a well-tested series of templates for automated outbreak and survey data analysis.
}

% \cventry{2018--2020}{Coordinator of Software Development}{}{R Epidemics Consortium (RECON)}{}{
%   Worked with software developers, mathematical modellers, and field epidemiologists to \textbf{design quality standards for software development, validation testing, and analysis of epidemiological data}.
% }




\section{Employment}

\cventry{2020--Present}{Lesson Infrastructure Developer}{The Carpentries}{}{}{
  Lead development and deployment of a modular infrastructure for building reproducible lessons to bring data science skills to researchers worldwide.
}
\cventry{2018--2020}{Research Softare Engineer}{Imperial College London}{}{}{
  Developed well-tested R packages for field epidemiology and modelling\\ (\textbf{4} peer-reviewed publications, \textbf{1} first-author)
}

\cventry{2017--2018}{Postdoctoral Researcher}{University of Nebraska-Lincoln}{}{}{
  Population genetics of the white mold pathogen \textit{Scleortinia sclerotiorum}\\ (\textbf{4} peer-reviewed publications, \textbf{2} first-author)
}

\cventry{2012--2016}{Graduate Research Assistant}{Oregon State University}{}{(Dissertation Research)}{
  Development and application of tools for genetic analysis of clonal populations\\ (\textbf{9} peer-reviewed publications, \textbf{4} first-author)
}

\section{Qualifications}

\cventry{2016}{Ph.D. Botany and Plant Pathology (3.65 GPA)}{Oregon State University (OSU)}{Corvallis, OR, USA}{}{
  Dissertation: Development and Application of Tools for Analysis of Clonal Populations
}
\cventry{2007}{B.S. Biology}{Truman State University (TSU)}{Kirksville, MO, USA}{}{}

\section{Selected Peer-reviewed Publications and Talks}

\begin{itemize}
  \item \textbf{Kamvar ZN} (2022) Building Accessible Lessons with R and Friends.
    \textit{rstudio::conf(2022)}: \url{https://zkamvar.netlify.app/talk/carpentries-rstudio-2022/}
  \item \textbf{Kamvar ZN}, Amaradasa BS, Jhala R, McCoy S, Steadman JR,
  Everhart SE. (2017) Population structure and phenotypic variation of
  \textit{Sclerotinia sclerotiorum} from dry bean (\textit{Phaseolus vulgaris})
  in the United States. \textit{PeerJ} \textbf{5}:e4152 doi: \href{https://doi.org/10.7717/peerj.4152}{10.7717/peerj.4152}\\
  % \rule[0.25\baselineskip]{0.25\textwidth}{0.5pt}\\
  data/analysis: \href{https://github.com/everhartlab/sclerotinia-366#readme}{https://github.com/everhartlab/sclerotinia-366}\\
  doi:\phantom{t/analysis:}
  \href{https://doi.org/10.17605/OSF.IO/EJB5Y}{10.17605/OSF.IO/EJB5Y}

  % \item \textbf{Kamvar ZN}, Tabima JF, Gr\"unwald NJ. (2014) \textit{Poppr}: an
  % R package for genetic analysis of populations with clonal, partially clonal,
  % and/or sexual reproduction. \textit{PeerJ} \textbf{2}: e281. doi: \href{https://doi.org/10.7717/peerj.281}{10.7717/peerj.281}
\end{itemize}

%----------------------------------------------------------------------------------------
%	WORK EXPERIENCE SECTION
%----------------------------------------------------------------------------------------
%\section{Selected Open Source Projects/Events}
%\cvitemwithcomment{Hackathons:}{\href{https://github.com/NESCent/r-popgen-hackathon/\#readme}{\#RPopHack (NESCent)}}{Improving the interoperability, scalability, and discoverability of population genetics packages in R}
%\cvitemwithcomment{}{\href{http://hackout3.ropensci.org/}{\#hackout3 (ROpenSci)}}{Creating open source tools for disease outbreak response}
%\cvitemwithcomment{Software:}{\href{https://github.com/grunwaldlab/poppr/\#readme}{poppr} (maintainer)}{R package for analysis of populations with mixed reproductive modes}
%\cvitemwithcomment{}{\href{https://github.com/thibautjombart/adegenet/\#readme}{adegenet} (contributor)}{R package for multivariate analysis of population genetics}
%\cvitemwithcomment{Education:}{\href{http://popgen.nescent.org}{http://popgen.nescent.org}}{Community-driven educational resource for population genetics in R}


%%----------------------------------------------------------------------------------------
%%	TEACHING EXPERIENCE SECTION
%%----------------------------------------------------------------------------------------

%\section{Teaching}
%\cventry{2016}{\href{http://grunwaldlab.github.io/Reproducible-science-in-R/}{Reproducible Science R}}{Workshop}{APS}{}{}
%\cventry{Spring 2012}{Introductory Botany}{Botany and Plant Pathology Dept.}{OSU}{}{}
%\cventry{2014 \& 2015}{\href{http://grunwaldlab.github.io/Population_Genetics_in_R}{Population Genetics in R}}{Workshop}{OSU/APS}{}{}
%% \cventry{2012}{Introductory Biology}{Biology Dept.}{OSU}{}{}
%% \cventry{2008--2011}{English Instructor}{K--12}{Daegu}{South Korea}{}
%% \cventry{2006 \& 2007}{Cell Biology TA}{Biology Discipline}{TSU}{}{}

%%----------------------------------------------------------------------------------------
%%	OUTREACH SECTION
%%----------------------------------------------------------------------------------------
%% \section{Outreach and Service}

%% \cventry{2012--Present}{Active Contributor}{Bioinformatics Users Group}{OSU}{}{
%% 	Contributed presentations and discussions relevant to use of bioinformatics tools.
%% }

%%----------------------------------------------------------------------------------------
%%	LEADERSHIP SECTION
%%----------------------------------------------------------------------------------------
%\section{Leadership/Organizing}

%\cventry{2015--2016}{Social Chair}{\href{http://cge6069.org/}{Coalition of Graduate Employees}}{labor union}{Corvallis, OR}{}
%\cventry{2012--2014}{Treasurer}{\href{http://gsa.bpp.oregonstate.edu/}{Graduate Student Association}}{Department of Botany and Plant Pathology}{OSU}{}

%\section{Creative Community Projects}

%\cventry{2012--2016}{\href{http://blogs.oregonstate.edu/inspiration/about-inspiration-dissemination/}{Inspiration Dissemination}}{radio program}{co-creator/host/producer}{}{Mission: Telling the stories about the lives and research of graduate students}
%\cventry{2009-03-14}{\href{https://crapartdaegu.wordpress.com/2009/02/20/statement-of-purpose-\%EC\%B7\%A8\%EC\%A7\%80-\%EC\%84\%B1\%EB\%AA\%85/}{Crap Art: Daegu}}{charity art show}{}{co-creator/organizer/participant}{Mission: Community-driven, process-based, spontaneous art creation}
%%----------------------------------------------------------------------------------------
%%	AWARDS SECTION
%%----------------------------------------------------------------------------------------

%\section{Awards}

%% \cvitem{2014}{OSU Botany and Plant Pathology Anita Summers Travel Award -- \$1000}
%\cvitem{2014}{Most Innovative [Radio] Program -- Intercollegiate Broadcasting System}
%% \cvitem{2013}{Seattle Institute For Statistical Genetics Travel Award -- \$450}
%\cvitem{2013--Present}{Graduate Student Travel Awards -- \$3100}
%\cvitem{2003--2006}{Truman State University Scholarships/Research Stipend -- \$5000}
%% \cvitem{2006}{Truman State University Summer Research Stipend -- \$3000}
%% \cvitem{2003}{Truman State University Presidential Leadership Scholarship -- \$2000}

%----------------------------------------------------------------------------------------
%	COMMUNICATION SKILLS SECTION
%----------------------------------------------------------------------------------------
