

%----------------------------------------------------------------------------------------
%	EDUCATION SECTION
%----------------------------------------------------------------------------------------
\vspace{5pt}

I am a \textbf{highly-motivated software engineer} of several published packages for both
population genetics/genomics and epidemiology in the R programming language. As
a \textbf{domain expert in computational biology}, software engineer, and a science
communicator, I have both the specialist and generalist skillsets needed for
efficient communication and collaboration between team members with diverse
technical backgrounds.

\section{Technical Skills}

\cvitem{Curriculum:}{\textbf{5 years experience} in collaborative \textbf{open source} lesson development targeted towards \textbf{\mbox{interdisciplinary} and diverse audiences}}
\cvitem{}{Developed lessons for \textbf{9 workshops} that reached \textbf{>300 people} in international conferences}
\cvitem{}{Collaboratively developed \textbf{5 tutorial websites} in \textbf{RMarkdown and blogdown} to host \textbf{\mbox{discoverable} lessons for non-technical audiences}}
\cvitem{}{Trained in \textbf{evidence-based} undergraduate STEM teaching (CIRTL)}

\cvitem{Technology:}{Highly experienced (\textbf{8 years}) in \textbf{collaborative development with Git and GitHub}; managing teams and code review}
\cvitem{}{\textbf{Expert R developer} of several widely used R packages (poppr, aweek, incidence) (\textbf{8 years})}
\cvitem{}{Maintained \textbf{reproducible research compendium in Docker} (\textbf{2 years}) under continuous integration (Kamvar \textit{et al.} 2017)}
\cvitem{}{Experienced in creating unique \textbf{Python} 3 modules for simulation analysis pipeline (\textbf{3 years})}
\cvitem{}{Highly experienced with continuous integration (\textbf{Travis (6 years), Appveyor, and CircleCI}) for \textbf{automated testing} on multiple platforms}

\cvitem{Communication:}{Highly experienced in \textbf{quickly troubleshooting} and resolving \textbf{>200 forum questions} in user forums for the poppr and adegenet packages ($\sim$1 day turnaround)}
\cvitem{}{\textbf{Co-founder and host (5 years)} of award-winning \textbf{science communication radio program}, Inspriration Dissemination (\url{http://blogs.oregonstate.edu/inspiration/})}
\cvitem{}{Published \textbf{19 peer-reviewed publications} including \textbf{7 first-author publications}}

\section{Leadership}

\cventry{2018--2019}{Lead Developer}{\href{https://r4epis.netlify.com}{R4Epis Project}}{M\'{e}decins Sans Fronti\`{e}res (Doctors Without Borders)}{}{
  \textbf{Remote} collaboration with \textbf{diverse team of developers and field epidemiologists} to create a well-tested series of \textbf{templates and tutorials for automated outbreak and survey data analysis}.
}

\cventry{2018--Present}{Coordinator of Software Development}{}{R Epidemics Consortium (RECON)}{}{
  Worked with software developers, mathematical modellers, and field epidemiologists to \textbf{design quality standards for software development, validation testing, and analysis of epidemiological data}.
}

\section{Selected Workshops/Primers}

\cventry{2014--2015}{\url{https://grunwaldlab.github.io/Population_Genetics_in_R}}{}{}{}{3-hour workshop on R for population genetic analysis using the \textit{poppr} package.}
\cventry{2016--Present}{\url{https://popgen.nescent.org}}{}{}{}{\textbf{Peer-reviewed community-submitted tutorials} for population genetic analysis in R}
\cventry{2017}{\url{https://everhartlab.github.io/IntroR}}{}{}{}{Introduction to R for plant pathologists including data import, reshaping, and graphics}
\cventry{2018--Present}{\url{https://reconlearn.org/post/stegen}}{}{}{}{Introduction to R for epidemiologists including data cleaning, graphics, and descriptive analysis}

\section{Selected Projects (on GitHub)}

\cvitemwithcomment{R package:}{\href{https://grunwaldlab.github.io/poppr/}{grunwaldlab/poppr} (maintainer)}{analysis of populations with mixed reproductive modes}
% \cvitemwithcomment{}{\href{https://github.com/thibautjombart/adegenet/\#readme}{thibautjombart/adegenet} (contributor)}{multivariate analysis of population genetics}
% \cvitemwithcomment{}{\href{https://repidemicsconsortium/incidence}{reconhub/incidence} (maintainer)}{aggregation and visualization of disease incidence}
% \cvitemwithcomment{}{\href{https://repidemicsconsortium/aweek}{reconhub/aweek} (maintainer)}{easy conversion betweend date and week formats}
% \cvitemwithcomment{Analyses:}{\href{https://github.com/zkamvar/clonal-inference-simulations}{zkamvar/clonal-inference-simulations}}{simulation-based analysis in \textbf{Python}, \textbf{R}, and \textbf{BASH}}

% \cvitemwithcomment{}{\href{https://github.com/zkamvar/read-processing}{zkamvar/read-processing}}{variant discovery from 55 \textit{Sclerotinia sclerotiorum} genomes in \textbf{Make}}
\cvitemwithcomment{Analysis:}{\href{https://github.com/everhartlab/sclerotinia-366}{everhartlab/sclerotinia-366}}{\textbf{fully automated} and reproducible analysis in \textbf{Docker}}



\newpage


\section{Employment}

\cventry{2018--Present}{Research Softare Engineer}{Imperial College London}{}{}{
  Developed well-tested R packages for field epidemiology and modelling\\ (\textbf{4} peer-reviewed publications, \textbf{1} first-author)
}

\cventry{2017--2018}{Postdoctoral Researcher}{University of Nebraska-Lincoln}{}{}{
  Population genetics of the white mold pathogen \textit{Scleortinia sclerotiorum}\\ (\textbf{4} peer-reviewed publications, \textbf{2} first-author)
}

\cventry{2012--2016}{Graduate Research Assistant}{Oregon State University}{}{(Dissertation Research)}{
  Development and application of tools for genetic analysis of clonal populations\\ (\textbf{9} peer-reviewed publications, \textbf{4} first-author)
}

\section{Qualifications}

\cventry{2016}{Ph.D. Botany and Plant Pathology}{Oregon State University (OSU)}{Corvallis, OR, USA}{}{
  Dissertation: Development and Application of Tools for Analysis of Clonal Populations
}
\cventry{2007}{B.S. Biology}{Truman State University (TSU)}{Kirksville, MO, USA}{}{}

\section{Selected Peer-reviewed Publications}

\begin{itemize}
  \item \textbf{Kamvar ZN}, L\'opez-Uribe MM, Coughlan S, Gr\"unwald NJ, Lapp
  H, Manel S (2016). Developing educational resources for population genetics
  in R: an open and collaborative approach. \textit{Molecular Ecology Resources}.
  \textbf{17}:1 120-128 doi:
  \href{http://doi.org/10.1111/1755-0998.12558}{10.1111/1755-0998.12558}

  \item \textbf{Kamvar ZN}, Amaradasa BS, Jhala R, McCoy S, Steadman JR,
  Everhart SE (2017). Population structure and phenotypic variation of
  \textit{Sclerotinia sclerotiorum} from dry bean (\textit{Phaseolus vulgaris})
  in the United States. \textit{PeerJ} \textbf{5}:e4152 doi: \href{https://doi.org/10.7717/peerj.4152}{10.7717/peerj.4152}\\
  % \rule[0.25\baselineskip]{0.25\textwidth}{0.5pt}\\
    \textit{data/analysis:} \href{https://github.com/everhartlab/sclerotinia-366#readme}{https://github.com/everhartlab/sclerotinia-366}\\
    \textit{doi:\phantom{t/analysis:}}
  \href{https://doi.org/10.17605/OSF.IO/EJB5Y}{10.17605/OSF.IO/EJB5Y}

\end{itemize}

